README.md

```diff
@@ -195,4 +195,21 @@ result of the installation and the login processes respectivaly. They are meant
 particular to record the offline and online tokens in your app's database.
 They can be synchronous or asynchronous functions taking the storename and the token
 as arguments.
-```
+
+
+### Shopify Multipass
+[Shopify Multipass](https://shopify.dev/docs/admin-api/rest/reference/plus/multipass) <br>
+This helper class generates token or URL that's needed for Shopify Multipass login (Only available in Shopify Plus)
+```python
+from spylib import Multipass
+
+# Customer email is required to generate token or URL
+customer_data = {"email": "customer@email.com"}
+# Initiate the Multipass class
+multipass = Multipass("ShopifyMultipassToken")
```

Comment:
It's called a secret in Shopify's documentation so let's be consistent.
```suggestion
multipass = Multipass("MULTIPASSSECRET")
```

Reply:
I think it should be ready to go.